homework-14/main.js

```diff
@@ -0,0 +1,121 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const inputYear = document.querySelector('.year-input');
+    const resultCentury = document.querySelector('.result-century');
+    document.querySelector('.year-submit').addEventListener('click', (event) => {
+        event.preventDefault();
+        if (inputYear.value <= 0) {
+            resultCentury.innerHTML = `Result is't valid.`;
+        } else {
+            let century = Math.floor(inputYear.value / 100) + 1;
+            if (century % 10 === 0) {
+                century = century + 'th';
+                resultCentury.innerHTML = `Result: ${inputYear.value} is a ${century} century.`;
+            } else if (century % 10 === 1) {
+                century = century + 'st';
+                resultCentury.innerHTML = `Result: ${inputYear.value} is a ${century} century.`;
+            } else if (century % 10 === 2) {
+                century = century + 'nd';
+                resultCentury.innerHTML = `Result: ${inputYear.value} is a ${century} century.`;
+            } else if (century % 10 === 3) {
+                century = century + 'rd';
+                resultCentury.innerHTML = `Result: ${inputYear.value} is a ${century} century.`;
+            }
+        }
+    })
+
+    const userValues = document.querySelector('.value-input');
+    const resultValue = document.querySelector('.result-value');
+
+    function vowelOne(value) {
+        const vowels = 'aeiouy';
+        let result = '';
+
+        for (let symbol of value) {
+            if (vowels.includes(symbol)) {
+                result += '1';
+            } else {
+                result += '0';
+            }
+        }
+        return result;
+    }
+    document.querySelector('.value-submit').addEventListener('click', (event) => {
+        event.preventDefault();
+        if (userValues.value === '') {
+            resultValue.innerHTML = `Result is't valid.`;
+        } else {
+            resultValue.innerHTML = `Result: ${vowelOne(userValues.value)}.`;
+        }
+    })
+
+    const userValues3 = document.querySelector('.value3-input');
+    const resultReversedValue3 = document.querySelector('.result-value3');
+
+    function reversedValue(value) {
```

Comment:
`Назви ф-ій й виклик цих ф-ій має бути такий самий, як й у представлених прикладах`
```suggestion
    function spinWords(value) {
```